### DISCUSSION TITLE
Can't update to netbox 3.6.3

### DISCUSSION THREAD

**Discussion by Scheppers-wetteren** [netbox-community/netbox-docker#1094] [Getting Help]
I'm trying to update Netbox to version 3.6.3 (from 3.4), but I am unable to do so. I'm getting the following error;

ERROR: The Compose file './docker-compose.yml' is invalid because:
services.netbox-housekeeping.depends_on contains an invalid type, it should be an array
services.netbox-worker.depends_on contains an invalid type, it should be an array

**Comment by tobiasge** [comment]
The error message suggests that you are using an old docker-compose version. Please use the new docker compose plugin.

**Reply by Scheppers-wetteren** [reply]
Thanks for the reply!

Could you provide more details on what you mean by docker-compose version? I'm running Ubuntu 20.04.6 LTS.

I did the main install by following a guide online, my docker knowlegde is very limited.

Thanks!

**Reply by tobiasge** [reply]
Your docker-compose is too old. See here: https://docs.docker.com/compose/install/linux/
